lib/jazz_fingers/amazing_print.rb

```diff
@@ -1,22 +1,22 @@
 module JazzFingers
-  AWESOME_PRINT = {
+  AMAZING_PRINT = {
     indent: 2,
     sort_keys: true,
     color: {
       args: :greenish,
-      array: :pale,
+      array: :whiteish,
       bigdecimal: :blue,
```

Comment:
`:pale` is [unsupported](https://github.com/amazing-print/amazing_print#usage) — changed to `:whiteish`